### PR TITLE
Und 555

### PR DESCRIPTION
**Pull request by Ashish-srij**
und-555

### CODE REVIEW

File: stories/Organism/Text/PostHeader/PostHeader.jsx [hunk 1]
```diff
@@ -1,17 +1,15 @@
-import React from 'react';
-import './postheader.scss';
-import { Breadcrumbcomponent } from '../../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs';
-import { Post } from '../../../Molecules/Text/Post/Post';
+import React from "react";
+import "./postheader.scss";
+import { Breadcrumbcomponent } from "../../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs";
+import { Post } from "../../../Molecules/Text/Post/Post";
```
Comment (surbhi90) [inline stories/Organism/Text/PostHeader/PostHeader.jsx:4]: @Ashish-srij Please lint the code as it should have the single quotes

File: stories/Organism/Text/PostHeader/postheader.scss [hunk 2]
```diff
@@ -1,12 +1,10 @@
-@import '../../../assets/scss/variables';
-@import '../../../assets/scss/typography';
-@import '../../../assets/scss/mixins';
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/typography";
+@import "../../../assets/scss/mixins";
```
Comment (surbhi90) [inline stories/Organism/Text/PostHeader/postheader.scss:3]: @Ashish-srij Please lint the code as it should have the single quotes

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Organism/Text/PostHeader/postheader.scss (hunk 2 above)
Comment (Ashish-srij) [reply stories/Organism/Text/PostHeader/postheader.scss]: @surbhi90 done.

File: stories/Organism/Text/PostHeader/PostHeader.jsx (hunk 1 above)
Comment (Ashish-srij) [reply stories/Organism/Text/PostHeader/PostHeader.jsx]: @surbhi90 Done

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**